proglearn/tests/test_KNNClassificationVoter.py

```diff
@@ -41,3 +44,5 @@ def test_correct_vote(self):
         assert_allclose(Y_test, kcv.vote(X_test), atol=1e-4)
```

Comment:
X_test and Y_test do not have the same size.

Reply:
They don't, but Y_test and kcv.vote(X_test) have the same size.